program-test/src/lib.rs

```diff
@@ -813,14 +808,35 @@ impl ProgramTestContext {
 
         Self {
             banks_client,
-            block_commitment_cache,
-            payer,
             last_blockhash,
+            payer: genesis_config_info.mint_keypair,
+            genesis_config: genesis_config_info.genesis_config,
             bank_forks,
+            block_commitment_cache,
             _bank_task: bank_task,
         }
     }
 
+    /// Fake vote activity to accrue vote credits, and eventually stake rewards
+    pub fn simulate_vote_activity(&mut self, voter: &Pubkey, number_of_slots: u64) {
+        let bank_forks = self.bank_forks.read().unwrap();
+        let bank = bank_forks.working_bank();
+        let working_slot = bank.slot();
+
+        // generate some vote activity for rewards
+        let mut voter_account = bank.get_account(voter).unwrap();
+        let mut vote_state = VoteState::from(&voter_account).unwrap();
+
+        // This can slow down testing for huge warp amounts, so limit to 2 dev epochs
```

Comment:
"so limit to 2 dev epochs" -- where's this limit?  I don't see it in the code below

Reply:
Oops, that was from some testing, I can get rid of the comment

Reply:
done in `20f5dfd`